docs/simulation/calculations.yaml

```diff
@@ -92,4 +92,32 @@ POPs:
               owners: total size of owner POPs in state
               workers: number of employed workers in RGO
               max(0.5: Hardcoded minimum of half the RGO income.
-              2*: Hardcoded value.
\ No newline at end of file
+              2*: Hardcoded value.
+Military:
+  Reinforcements per army:
+    formula: 5 * round(supply factor * sum of unit reinforcements * REINFORCE_SPEED * local reinforce rate * (1 + reinforce_rate) * (1 + reinforce_speed) / 5)
+    supply factor:
+      if unit received all its supplies: 1
+      else: complex graph, see https://www.desmos.com/calculator/gu1zpo01qz
+    unit reinforcements: 1000 * max_strength * unit reinforce rate
+      max_strength: as defined in /units/<unit>.txt
+      1000: (max_)strength is defined in thousands of soldiers. Reinforcements per army uses the number of soldiers.
+      unit reinforce rate:
+        if unit is mobilised: 0.5
+        else if pop is too small for a single unit: 0.5
+        else if pop supports too many units: number of units it can support / number of supported units
+        else: 1
+    REINFORCE_SPEED: defines.military.REINFORCE_SPEED
+    local reinforce rate:
+      if location owner == army owner: 2
+      else if has military access (via treaty, occupation or allies in war): 1
+      else if is exiled: 0
+      else if location controller is hostile:
+        if location neighbours a province controller by an ally in this war: 0.5
+        else if location is coastal & not blockaded: 0.25
+        else: 0.1
+      else if location is uncolonised: 0
+      else: no clue
+    reinforce_rate: sum of reinforce_rate modifiers from tech
+    reinforce_speed: sum of reinforce_speed modifiers on country
+    5 * round() / 5: round to multiple of 5
```

Comment:
Shouldn't the `/ 5` be inside the `(brackets)`? Also is this rounding to the nearest integer, so `round(0.4) = 0`, `round(0.5) = 1`?

Reply:
Yes. Thankfully the formula has the correct variant. I simply messed up in the notes.